docker-compose.yml

```diff
@@ -21,7 +21,9 @@ services:
     depends_on:
     - database
   scrape:
-    image: openstates/openstates-pupa
+    build:
```

Comment:
I'm guessing this was for testing local changes--did you want this change to be part of the patch?